pkg/model/components/discovery.go

```diff
@@ -61,6 +61,11 @@ func (b *DiscoveryOptionsBuilder) BuildOptions(o interface{}) error {
 				if err != nil {
 					return err
 				}
+			case *vfs.GSPath:
+				serviceAccountIssuer, err = base.GetHTTPsUrl(clusterSpec.IsIPv6Only())
```

Comment:
Not a blocker, but we should use an interface and cast instead of using this switch.

Reply:
I agree, I just wasn't sure yet what it should look like - and the PR was already getting big...